test/lib/specHelperSpec.js

```diff
@@ -0,0 +1,111 @@
+import ActionPrototype from '../../src/lib/ActionPrototype'
+import StorePrototype from '../../src/lib/StorePrototype'
+import Dispatcher from '../../src/lib/ep-dispatcher'
+import assign from 'object-assign'
+import KeyMirror from 'keymirror'
+//import * as FUHelpers from '../support/ComponentHelpers'
+import React from 'react/addons'
+import TestRig from '../../src/testUtils/TestRig.react'
+
+var TestUtils = React.addons.TestUtils;
+
+var TestAction = assign(ActionPrototype, {
+  Types: KeyMirror({
+    GOT_THING: null,
+    NOTHING: null
+  }),
+  fetchThing: function() {
+    //actually fire API
+    Dispatcher.dispatch({
+      actionType: TestAction.Types.GOT_THING,
+      data: {foo: 'bar'}
+    })
+
+  },
+  fetchWithWrongDispatch: function() {
+    Dispatcher.dispatch({
+      actionType: TestAction.Types.NOTHING,
+      data: {foo: 'bar'}
+    })
+  }
+})
+
+var TestStore = StorePrototype(TestAction.Types.GOT_THING)
+
+var TestComponent = React.createClass({
+  getInitialState: function() {
+    return {
+      value: 'initial value'
+    };
+  },
+  handleSubmit: function() {
+    var value = $(this.refs.value.getDOMNode()).val()
+    this.setState({value: value})
+  },
+  render: function() {
+    return (
+      <div><h1>Test Component</h1>
+        <div className='answer'>Form Value: {this.state.value}</div>
+        <input ref='value' defaultValue={this.state.value}/>
+        <a onClick={this.handleSubmit}>Submit</a>
+      </div>
+    )
+  }
+})
+
+describe("Fluxxed up test helpers", function() {
+
+  describe("Action helpers", function() {
+    it("dispatches to test store", function(done) {
+      expect(TestAction.fetchThing).to.eventually(done).informRegisteredStore(TestStore)
+    })
+
+    it("wrong action will not notify the store", function(done) {
+      expect(TestAction.fetchWithWrongDispatch).to.not.eventually(done).informRegisteredStore(TestStore)
+    })
+  })
+
+  describe("Component Helpers", function() {
+    var node, domNode, context
+
+    beforeEach(()=> {
+      //todo either expose a method for this. OR -- use old school way.
```

Comment:
@BJK I would lean toward exposing a method since the code will always be identical and it will be another step toward minimizing boilerplate--no clarity will be lost, and we can use ES6 destructuring to define all three variables in one statement.